get-config.js

```diff
@@ -0,0 +1,21 @@
+function readConfig() {
+	try {
+		return require( './config' )();
+	} catch ( e ) {
+		console.error(
+			'Could not read in the required configuration file.\n' +
+			'This file should exist as `config.js` inside the project root directory.\n' +
+			'Please consult the project README.md for further information.\n'
+		);
+		process.exit( 0 );
```

Comment:
@beaucollins I went ahead and instructed the process to exit here since the app won't boot without the config message.